app/assets/javascripts/page.js

```diff
@@ -0,0 +1,138 @@
+"use strict";
```

Comment:
Just want to note that the inclusion of Backbone here made this file really easy to read and understand. Nice job!